examples/layertree.js

```diff
@@ -25,11 +25,6 @@ var app = {};
 app.module = angular.module('app', ['ngeo']);
 
 
-// Use the default "popup" template.
-app.module.value('ngeoPopupTemplateUrl',
```

Comment:
forget it, it was just that before the override of this value was useless cause it's passed trhough attributes